EU4toV2/Source/V2World/V2Country.cpp

```diff
@@ -641,35 +643,81 @@ void V2Country::initFromEU4Country(
 	// Government
 	government = governmentMapper::matchGovernment(srcCountry->getGovernment());
```

Comment:
Since you're modifying this, it (and other such blocks) should be put into functions instead of being part of a mega-function.

Perhaps determineGovernment(), applyIdeaEffects(), determineUpperHouse(), determineLiteracy(), determineTechSchool()

Reply:
split megafunction into 10 sensible smaller ones.